### DISCUSSION TITLE
Llava style model support in Onnx?

### DISCUSSION THREAD

**Discussion by kinchahoy** [microsoft/onnxruntime-genai#369] [Model support]
Hey folks,

I was thinking about training a small Phi-3 + OpenClip Llava style model (building off the MobileVLM work). Do you plan to support that sort of multi-modal model with Onnx (or ideally onnxruntime-genai)? Let me know if I can help test etc.

**Comment by natke** [comment]
@kinchahoy Yes we do have this support on our road map. I am moving to our Discussions forum to track it there
